yarn.lock

```diff
@@ -17892,6 +17887,35 @@ store2@^2.7.1:
   resolved "https://registry.yarnpkg.com/store2/-/store2-2.10.0.tgz#46b82bb91878daf1b0d56dec2f1d41e54d5103cf"
   integrity sha512-tWEpK0snS2RPUq1i3R6OahfJNjWCQYNxq0+by1amCSuw0mXtymJpzmZIeYpA1UAa+7B0grCpNYIbDcd7AgTbFg==
 
+storybook-chromatic@^2.2.2:
```

Comment:
This one too 😬 Might be a good idea to check out the lock file from master and run `yarn install` to regenerate this with your dependencies